### PR TITLE
Fix for mitx

### PR DESCRIPTION
**Pull request by AnhadSawhney**
MITx has images with black diagrams on a transparent background. When the colors get inverted, it becomes black diagrams on a dark background, so they are not visible. This sets the background of the images to white.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -10109,6 +10109,16 @@ INVERT
 
 ================================
 
+*.mitx.*
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10112]:
```suggestion
mitx.*
```

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -10109,6 +10109,16 @@ INVERT
...
+
+CSS
+img {
+	border-color: currentcolor;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10116]: Is this needed?

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -10109,6 +10109,16 @@ INVERT
...
+	background: rgb(255, 255, 255) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:10117]:
```suggestion
	background-color: white !important;
```

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (AnhadSawhney) [reply src/config/dynamic-theme-fixes.config:10112]: The site is https://lms.mitx.mit.edu

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (AnhadSawhney) [reply src/config/dynamic-theme-fixes.config:10116]: I think this is the selector that works to find the images that are problematic

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:10112]: Dark reader already by default match all subdomains.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:10116]: But why changing the border color?

Comment (AnhadSawhney) [reply src/config/dynamic-theme-fixes.config:10116]: Ok seems it works without that. Sorry, I don't know anything about CSS, I just tried a couple of things until one worked.

**Comment by AnhadSawhney** [comment]
Should be good to merge now? @Gusted

**Review by Gusted — APPROVED** [review]

**Pull request merged**